Hestia/app/src/main/java/hestia/UI/elements/DeviceBar.java

```diff
@@ -58,7 +62,8 @@ public void initView() {
         switc.setVisibility(View.INVISIBLE);
 
         for(final Activator activator : device.getActivators()){
-            if(activator.getRank() == 0){
+            int toggleRank = Integer.valueOf(context.getResources().getString(R.string.toggleRank));
```

Comment:
Good change, but since many branches are currently working on refactoring the GUI code make sure it doesn't lead to merge conflicts